dataRetriever/txpool/shardedTxPool.go

```diff
@@ -12,6 +13,9 @@ import (
 	"github.com/ElrondNetwork/elrond-go/storage/txcache"
 )
 
+var _ counting.Countable = (*shardedTxPool)(nil)
```

Comment:
Are these variables useful to something. If yes, please add comments.

Reply:
It forces the compiler to show an error when interface not implemented.
We will apply these for each interface implementation in the project (wrt. to the audit).

Reply:
Ok